docs/se/webclient/introduction.adoc

```diff
@@ -233,4 +245,77 @@ requestBuilder.readerContext().registerReader(JsonSupport.reader()); // <2>
 requestBuilder.request(JsonObject.class)
 ----
 <1> Adds JSON-P writer only to this request.
-<2> Adds JSON-P reader only to this request.
\ No newline at end of file
+<2> Adds JSON-P reader only to this request.
+
+=== WebClient TLS setup
+
+Configure TLS either programmatically or by the Helidon configuration framework.
+
+==== Configuring TLS in your code
+
+The one way to configure TLS in WebClient is in your application code.
```

Comment:
One way to configure TLS in WebClient is in your application code as shown below.